source/includes/examples/tf-example-backup-variables.rst

```diff
@@ -0,0 +1,21 @@
+.. code-block:::
+
+   variable "org_id" {
+     description = "Atlas organization ID"
+     type        = string
+   }
+
+   variable "project_name" {
+     description = "Atlas project name"
+     type        = string
+   }
+
+   variable "project_id" {
```

Comment:
in the backup example a project is being created so no need to pass a project_id